jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudformation/CloudFormationTemplate.kt

```diff
@@ -62,6 +64,13 @@ interface Parameter : NamedMap {
     fun constraintDescription(): String?
 }
 
+interface Variable {
```

Comment:
nit (naming): We seem to use two terms to refer to these `Variable` and `EnvironmentVariable` - we should pick one and be consistent.